### PR TITLE
Update to Use Toolkit v4

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
Update to use toolkit v4 and move toolkit to be a peer dependency.

### CODE REVIEW

File: config/wdio/wdio.conf.js [hunk 1]
```diff
@@ -13,10 +13,11 @@ const config = {
   axe: {
     inject: true,
     options: {
-      rules: [{
-        id: 'landmark-one-main',
-        enabled: false,
-      }],
+      rules: [
```
Comment (emilyrohrbough) [inline config/wdio/wdio.conf.js]: Added additional rules for axe MVB in toolkit v4

File: package.json [hunk 2]
```diff
@@ -92,15 +92,14 @@
     "terra-markdown": "^2.0.0",
     "terra-slide-panel": "^2.0.1",
     "terra-theme-provider": "^2.0.0",
-    "terra-toolkit": "^3.0.0",
     "to-ast": "^1.0.0",
-    "webpack-merge": "^4.1.2",
```
Comment (emilyrohrbough) [inline package.json]: This is a dependency included in toolkit

Comment (mjhenkes) [reply package.json]: We should still include it as a dependency here because we use it directly.

File: webpack.config.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (mjhenkes) [inline webpack.config.js]: This is a red flag 🎏

File: package.json (hunk 2 above)
Comment (emilyrohrbough) [reply package.json]: Does this mean we should not have included webpack-merge as a dependency of toolkit? it was included as a dependency in [v3 as well](https://github.com/cerner/terra-toolkit/pull/113/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R97)

But I will go ahead and make this change.

**Pull request merged**